solutions/ipetek/turtlesim_mouse_tracker.cpp

```diff
@@ -0,0 +1,92 @@
+#include<ros/ros.h>
+#include<geometry_msgs/Twist.h>
+#include<geometry_msgs/Point.h>
+#include<turtlesim/Pose.h>
+#define _USE_MATH_DEFINES
+#include <cmath>
+#include <math.h>
+#include<stdio.h>
+
+
+int SCREEN_X = 1920; //EDIT
+int SCREEN_Y = 1080; //EDIT
+int LINEAR_SPEED = 5;
+int ANGULAR_SPEED = 10;
+
+class MouseTurtleController{
+    private:
+        ros::NodeHandle node;
```

Comment:
convention to have private variables have `_` at the end is the default for ROS cpp applications
so
`ros::NodeHandle node_`